railties/lib/rails/generators/app_base.rb

```diff
@@ -377,7 +377,7 @@ def bundle_command(command)
         say_status :run, "bundle #{command}"
 
         # We are going to shell out rather than invoking Bundler::CLI.new(command)
-        # because `rails new` loads the Thor gem and on the other hand bundler uses
+        # because +rails new+ loads the Thor gem and on the other hand bundler uses
```

Comment:
Actually this is a comment in the method, not a doc. It is no worth to change to rdoc format.